ui/src/app/pages/data/cohort-review/detail-tab-table.component.tsx

```diff
@@ -287,7 +291,7 @@ interface State {
   start: number;
   sortField: string;
   sortOrder: DataTableSortOrderType;
-  expandedRows: Array<any>;
+  expandedRows: Array<any> | DataTableExpandedRows;
```

Comment:
Added type of expandedRows to match the return type of DataTable's onRowToggle.

Reply:
I don't believe so, because DataTableRowToggleParams expects a key of data and a value of Array<any> | DataTableExpandedRows.
![image](https://github.com/all-of-us/workbench/assets/24514630/2c340dd6-94e9-4065-b78e-949bc0ad8a7e)